airflow/lineage/backend/__init__.py

```diff
@@ -16,13 +16,12 @@
 # specific language governing permissions and limitations
 # under the License.
 #
-"""Sends lineage metadata to a backend"""
+"""Send lineage metadata to a backend"""
 
 
 class LineageBackend:
-    """Sends lineage metadata to a backend"""
-    def send_lineage(self,
-                     operator=None, inlets=None, outlets=None, context=None):
+    """Sends lineage metadata to a backend."""
```

Comment:
s here as well.

Reply:
Hi @potiuk, thank you for stepping in, I've changed here too, from:
```
def send_lineage(self,
                     operator=None, inlets=None, outlets=None, context=None):
```
to
```
def send_lineage(self, operator, inlets, outlets, context):
```

Please let me know if for you both (@feluelle and @potiuk) the changes are fine. Thanks a lot for your time! :)